### DISCUSSION TITLE
Problems with "StimulationProtocol"

### DISCUSSION THREAD

**Discussion by ShaBeNou** [expyriment/expyriment#220] [General]
Hi!

My name is Shahar and I'm looking into expyriment as a possible alternative to psychopy.

As I'm using fMRI I looked at the exampl at ["examples/fmri/motor_imagery"](https://github.com/expyriment/expyriment-stash/tree/master/examples/fmri/motor_imagery) but even though I have Expyriment version 0.10.0 installed I got the following error:

Traceback (most recent call last):
  File "motor_imagery.py", line 8, in module
    from expyriment.design.extras import StimulationProtocol
ImportError: cannot import name 'StimulationProtocol'

Thanks in Advance,
Shahar

**Comment by fladd** [comment]
Hi Shahar,

did you install the extras already, as described [here](https://docs.expyriment.org/Plugins.html)?

Best,
Florian

**Comment by ShaBeNou** [comment]
Didn't expect to get a reply so soon, thank you!

Your link helped me, apparently I needed to add the download from stash command - 

`expyriment.misc.download_from_stash(content="extras")`

since I downloaded expyriment with the [all] field I thougt it means the extras are already included :)

Thanks again!

**Comment by fladd** [comment]
Yeah, the [all] is about package dependencies. We could probably make that clearer in the documentation. Will make a note for it, thanks!
